presto-verifier/src/main/java/com/facebook/presto/verifier/Validator.java

```diff
@@ -446,6 +446,10 @@ private static boolean isPrestoQueryInvalid(SQLException e)
                 if (object instanceof Array) {
                     object = ((Array) object).getArray();
                 }
+                if (object instanceof byte[]) {
+                    byte[] varbinary = (byte[]) resultSet.getObject(i);
```

Comment:
You can leave this line out, and just cast `object` as a `byte[]`. However, I don't think converting this into a `String` is going to work, as you might have bytes that aren't legal characters. Instead I would convert it to a `SqlVarbinary` like `object = new SqlVarbinary((byte[]) object)`